File: Sakila_Net/VB_SakilaLib/Neg/NegCountry.vb
Imports  SakilaLib.Cl.Harrys.Sakila.DAO
Imports  SakilaLib.Cl.Harrys.Sakila.Model
Namespace Cl
    Namespace Harrys
        Namespace Sakila
            Namespace Neg

                Public Class ClNegCountry
                    Public Shared Function listar(nPagina as Integer ,cantidad as Integer) as DataTable
                        Return DAO.DAOCountry.SelectAll(nPagina, cantidad)	   
                    End Function
                    Public  Shared Function consultar( country as ClCountry) as Boolean
                        Return DAO.DAOCountry.SelectOne(country)     
                    End Function
                    Public  Shared Function crear( country as ClCountry) as Integer
                        Return DAO.DAOCountry.SqlInsert(country)     
                    End Function
                    Public  Shared Function eliminar( country as ClCountry) as Integer
                        Return DAO.DAOCountry.SqlDelete(country)     
                    End Function
                    Public  Shared Function actualizar( country as ClCountry) as Integer
                        Return DAO.DAOCountry.SqlUpdate(country)     
                    End Function
                End Class

            End Namespace  
        End Namespace  
    End Namespace  
End Namespace  

File: Sakila_Net/CS_SakilaLib/Neg/NegCountry.cs
using System.Data;
using Cl.Harrys.Sakila.DAO;
using Cl.Harrys.Sakila.Model;
namespace Cl.Harrys.Sakila.Neg
{
    public class ClNegCountry
    {
        public static DataTable listar(int nPagina ,int cantidad)
        {
            return DAO.DAOCountry.sqlSelectAll(nPagina, cantidad);
        }
        public  static bool consultar( ClCountry country) 
        {
            return DAO.DAOCountry.sqlSelect(country);
        }
        public  static int crear(ClCountry country)
        {
            return DAO.DAOCountry.sqlInsert(country);
        }
        public  static int eliminar(ClCountry country)
        {
            return DAO.DAOCountry.sqlDelete(country);
        }
        public  static int actualizar(ClCountry country)
        {
            return DAO.DAOCountry.sqlUpdate(country);
        }
    }
}
